resources/qml/Widgets/CuraProgressBar.qml

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2019 Ultimaker B.V.
```

Comment:
Consider moving this to Uranium.

Reply:
I think this can be moved to Uranium.

Reply:
This is made more generic in https://github.com/Ultimaker/Cura/pull/5470 and we can consider moving it there

Reply:
#5470 was merged but it's not yet moved to Uranium.

Reply:
I moved the file over.